### DISCUSSION TITLE
App signing (maybe offtopic)

### DISCUSSION THREAD

**Discussion by BobbyESP** [JunkFood02/Seal#332] [Q&A]
I know this isn't related to Seal, but i liked to ask it here. I wanted to know if you had to fill out a form for entering the Play Protect whithelist. Thank you!

**Comment by JunkFood02** [comment]
I don't even have a Play developer account 🤔

**Reply by BobbyESP** [reply]
It's not necessary. Im asking it because an app that I'm developing is marked as malicious kekw
